project/plugins.sbt

```diff
@@ -25,7 +25,7 @@ addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.3")
 addSbtPlugin("net.vonbuchholtz" % "sbt-dependency-check" % "0.2.0")
 
 // dependency tracker plugin - needed for snyk cli integration
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0")
+addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.10.0-RC1")
```

Comment:
we are on sbt1.3.2 now but there are bugs with its interaction with this plugin.  this version of sbt dependency graph will appear to work rather than crash https://github.com/jrudolph/sbt-dependency-graph/issues/178#issuecomment-514529823
however it doesn't always produce the right result https://github.com/coursier/coursier/issues/1375
so we will need another version bump at some point, but this is just a small helper tool rather than something from the build itself.
Another option is just to remove it, and people can add it to their local ~/.sbt,  plugins.sbt file if they like to use it.